app/src/helpers/chain.ts

```diff
@@ -1,18 +1,10 @@
-import { harmonyOne } from '@wagmi/core/chains'
-import { type Chain } from '@wagmi/core'
+import { harmonyOne, sepolia } from 'wagmi/chains'
 
-const harmonyChain: Chain = {
-  ...harmonyOne,
-  rpcUrls: {
-    public: { http: ['https://rpc.ankr.com/harmony'] },
-    default: { http: ['https://rpc.ankr.com/harmony'] }
-  }
-}
+const chain = import.meta.env.VITE_TEST ? sepolia : harmonyOne
```

Comment:
default `harmonyOne` uses ankr RPC, which is dysfunctional most of the time. We should override it - make RPC URL a parameter loaded from env

Reply:
Resolved in https://github.com/polymorpher/otc-country/pull/23